config/src/genesis.rs

```diff
@@ -43,3 +43,48 @@ impl Default for ConfigurationProxy {
         }
     }
 }
+
+#[cfg(test)]
+pub mod tests {
+    use iroha_crypto::KeyPair;
+    use proptest::prelude::*;
+
+    use super::*;
+
+    /// Key-pair used by default for test purposes
+    #[allow(clippy::expect_used)]
+    fn placeholder_keypair() -> KeyPair {
+        let public_key = "ed01204cffd0ee429b1bdd36b3910ec570852b8bb63f18750341772fb46bc856c5caaf"
+            .parse()
+            .expect("Public key not in mulithash format");
+        let private_key = PrivateKey::from_hex(
+            iroha_crypto::Algorithm::Ed25519,
+            "d748e18ce60cb30dea3e73c9019b7af45a8d465e3d71bcc9a5ef99a008205e534cffd0ee429b1bdd36b3910ec570852b8bb63f18750341772fb46bc856c5caaf"
+        ).expect("Private key not hex encoded");
+
+        KeyPair::new(public_key, private_key).expect("Key pair mismatch")
+    }
+
+    #[allow(clippy::option_option)]
+    fn arb_keys() -> BoxedStrategy<(Option<PublicKey>, Option<Option<PrivateKey>>)> {
+        let (pub_key, _) = placeholder_keypair().into();
+        (
+            prop::option::of(Just(pub_key)),
+            prop::option::of(Just(None)),
+        )
+            .boxed()
+    }
+
+    prop_compose! {
+        pub fn arb_proxy()
+            (
+                (account_public_key, account_private_key) in arb_keys(),
+                wait_for_peers_retry_count_limit in prop::option::of(Just(DEFAULT_WAIT_FOR_PEERS_RETRY_COUNT_LIMIT)),
```

Comment:
Just like above, I'd consider adding warnings (but in a separate ticket).

Reply:
Created https://github.com/hyperledger/iroha/issues/2801